src/jquery-sticky-header-footer.js

```diff
@@ -60,8 +61,17 @@
                 /**
                     Remove added DOM elements and plugin data
                  */
-                $('.' + classNames.outerWrapper).before(element).remove();
-                element.removeData('plugin_' + pluginName);
+                $.each(['footerElement', 'headerElement'], function(idx, val) {
+                    var element = that[val];
+                    if (element) {
+                        if (element.isStuck) {
+                            that.unstick.call(that, element);
```

Comment:
Ensure that any nodes that may have been swapped out are properly returned to their starting positions.